qualtran/bloqs/block_encoding/lcu_block_encoding.py

```diff
@@ -227,6 +285,22 @@ def _extract_soqs(bloq: Bloq) -> Dict[str, 'SoquetT']:
         soqs |= bb.add_d(self.prepare.adjoint(), **_extract_soqs(self.prepare.adjoint()))
         return soqs
 
+    def wire_symbol(self, reg: Optional[Register], idx: Tuple[int, ...] = tuple()) -> 'WireSymbol':
+        if reg is None:
+            return Text('')
+        if reg.name == 'control':
+            return Circle(filled=bool(self.control_val))
+        else:
+            return TextBox('B[H]')
+
+    def get_single_qubit_controlled_bloq(self, control_val: int) -> 'LCUBlockEncodingZeroState':
```

Comment:
this should probably raise an error if `self.control_val` is already not `None`.

Reply:
Done.